index.js

```diff
@@ -1046,4 +1137,14 @@ let game = null;
     window.addEventListener('focus', event => {
         start = performance.now() - 1000 / 60;
     });
+
+    window.addEventListener("orientationchange", (event) => {
+        const angle = Math.abs(event.target.screen.orientation.angle);
+        if (angle === 90 || angle === 270) {
+            document.body.requestFullscreen()
+                .catch(error => console.error(`${error.message}. API can only be initiated after user gesture.`));
```

Comment:
Is this important for user information? Maybe we should display it to them via the in-game text?

Reply:
Added message that asks gamer to tap to enter full screen mode in case of an error.
That way game makes second attempt, but this time full screen mode was directly initiated by user gesture.